### PR TITLE
Add Evaluation APIViews & minor fix at Manager/TableView

### PR DESCRIPTION
**Pull request by holenet**
### Added Evaluation APIViews to TTRS App
- EvaluationList
- EvaluationDetail
- EvaluationLikeIt

For the specification, please refer to the [WIKI](https://github.com/SWPP/ttrs-backend/wiki/RESTful-API-Specification%3A-Evaluation)

### Minor fix TableView of Manager App
- made be able to support json request

### CODE REVIEW

**Comment by holenet** [comment]
At the `Evaluation` model, the `like_it` field should not be an `IntegerField` because an evaluation should not receive several *like it*'s from same student.
So the `like_it` field might have to be a `ManyToManyField` with `Student` model so that we can check duplication of students.

I close this pull request for now, please reply your opinion.

**Comment by lameinthebox** [comment]
Agree.

**Comment by holenet** [comment]
I implemented based on changing the type of the 'like_it' field in the Evaluation model.

File: backend/ttrs/serializers.py [hunk 1]
```diff
@@ -5,14 +5,22 @@
 from rest_framework.exceptions import ValidationError
 from django.core.exceptions import ValidationError as DjangoValidationError
 
-from .models import Student, College, Department, Major, Course, Lecture, TimeTable
+from .models import Student, College, Department, Major, Course, Lecture, Evaluation, TimeTable
 
 
 class StudentSerializer(serializers.ModelSerializer):
```
Comment (holenet) [inline backend/ttrs/serializers.py:11]: I reordered Meta class and several methods for `StudentSerializer`, in order of Meta class, get field methods and validation methods.

**Review by lameinthebox — APPROVED** [review]
Well done!

**Pull request merged**